editor/js/codes.js

```diff
@@ -80,6 +93,12 @@ function unload(codes) {
   });
 }
 
+/**
+ * Fetches a single automation_code section from the form and adds it to
```

Comment:
Is it worth noting which single code this is? (we do describe this in the node param description, so it's not necessary)

Reply:
Yeah, I think we could add "specified by `node`" but that feels redundant. Do you agree?

Reply:
Yep that's fair -- as-is is good with me.